dropwizard-jersey/src/main/java/io/dropwizard/jersey/params/DurationParam.java

```diff
@@ -0,0 +1,20 @@
+package io.dropwizard.jersey.params;
+
+import io.dropwizard.util.Duration;
+
+public class DurationParam extends AbstractParam<Duration> {
```

Comment:
Ideally, this may have been split into another PR, but that's ok here.

Can you write a test for this?